package.json

```diff
@@ -48,5 +49,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "dependencies": {
+    "docusaurus": "^1.14.7"
```

Comment:
Is this the old version?

Reply:
@phated  Actually it is recent version. https://www.npmjs.com/package/docusaurus
library with @docusaurus has updated to 2.4.1

Reply:
This package is old and should be deprecated. The new package is https://www.npmjs.com/package/@docusaurus/core which you've updated